source/codegen/metadata_mutation.py

```diff
@@ -41,7 +41,7 @@ def mark_non_proto_params(parameters):
        Their values should be derived from other sources in the service handlers."""
     for param in parameters:
         mechanism = common_helpers.get_size_mechanism(param)
-        if mechanism in {'len', 'ivi-dance'}:
+        if mechanism in {'len', 'ivi-dance', 'ivi_dance_with_a_twist'}:
```

Comment:
Can we have it be `ivi-dance-with-a-twist` instead? This will make it consistent with other size mechanisms like `ivi-dance` and `passed-in`.

Reply:
I changed it to `ivi-dance-with-a-twist`